src/main/java/concurrency/app/SumForkJoinCounter.java

```diff
@@ -0,0 +1,36 @@
+package concurrency.app;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.concurrent.ForkJoinTask;
+import java.util.concurrent.RecursiveTask;
+
+public class SumForkJoinCounter extends RecursiveTask<Long> {
+    private static final int THRESHOLD = 10;
```

Comment:
```suggestion
    private static final int THRESHOLD = 1000;
```
я думаю тут краще буде трохи підняти threshold

Reply:
done